src/en-GB/community/tech-talks/index.html

```diff
@@ -3,52 +3,89 @@
+        <li><strong>Where</strong>: Blue Jeans teleconference</li>
+        <li><strong>Recordings</strong>: All calls will be published to YouTube.</li>
+        <li>
+          <strong>Mailing list</strong>: to receive announcements, watch the
+          <a class="a" href="https://lists.ceph.io/postorius/lists/ceph-users.ceph.io/" target="_blank" rel="noreferrer noopener"
+            >ceph-users mailing</a
+          >.
+        </li>
+        <li>
+          <strong>Calendar</strong>:
+          <a
+            class="a"
+            href="https://calendar.google.com/calendar/b/1?cid=OXRzOWM3bHQ3dTF2aWMyaWp2dnFxbGZwbzBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ"
+            target="_blank"
+            rel="noreferrer noopener"
+            >Ceph Community</a
+          >
+          |
+          <a
+            class="a"
+            href="https://calendar.google.com/calendar/ical/9ts9c7lt7u1vic2ijvvqqlfpo0%40group.calendar.google.com/public/basic.ics"
+            target="_blank"
+            rel="noreferrer noopener"
+            >iCal</a
+          >
+        </li>
+      </ul>
+      <h2 class="h2">Join the stream</h2>
+      <p class="mb-0 p">
+        Or submit a request to host one of your own by emailing foundation AT ceph DOT io
+      </p>
+    </div>
+    <div>
+      <div class="aspect-ratio aspect-ratio--16x9">
+        <iframe
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/vQF17UBU4RE"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
     </div>
   </div>
-</div>
+</section>
 
 {% for item in tech_talks | chunkByYear | objectValues | reverse %}
-<h3>{{ item.name }}</h3>
-<ul>
-  {% for item in item.results %}
-  <li>
-    <time datetime="{{ item.date }}">{{ item.date | formatDate(locale) }}</time>
-    <a href="{{ item.url }}" target="_blank" rel="noreferrer noopener">{{ item.title }}</a> [{{ item.presenter }}] {% if item.date |
-    futureDate %}<span class="text-semibold">[Coming soon]</span>{% endif %}
-  </li>
-  {% endfor %}
-</ul>
+<section class="bg-grey-300 {% if loop.index0 %}pt-0{% endif %} section section--full">
+  <div class="wrapper">
+    <div class="max-w-256">
+      <h3 class="h3">{{ item.name }}</h3>
+      <table class="table">
+        <tbody>
+          {% for item in item.results %}
+          <tr>
+            <td class="w-36">
+              <time datetime="{{ item.date }}">{{ item.date | formatDate(locale) }}</time>
+            </td>
+            <td>
+              {% if item.url %}
+              <a class="a" href="{{ item.url }}" target="_blank" rel="noreferrer noopener">{% endif %}{{ item.title }}{% if item.url %}</a
+              >{% endif %} {% if item.presenter %}<span class="inline-block text-lower">{{ 'with' | i18n }} {{ item.presenter }}</span>{%
```

Comment:
@Pete-Robelou Is there a reason we `text-lower`'d this? The `tech_talks` content (data) is properly cased with names etc. Perhaps we render as is?

Reply:
It was to lowercase the word `with` as grammatically the whole line reads as a sentence. I guess we could just wrap a `span` around the word `with` so only that is lowercase ?

<img width="561" alt="Screenshot 2021-06-23 at 11 07 54" src="https://user-images.githubusercontent.com/13508865/123079086-807bdb80-d413-11eb-92da-49489a47e441.png">
<img width="570" alt="Screenshot 2021-06-23 at 11 08 07" src="https://user-images.githubusercontent.com/13508865/123079088-81147200-d413-11eb-9e61-5a891a13bbaa.png">

Reply:
Ah, gotcha. Makes sense. Yeah, good shout. Let's just apply that to `with` 👍

Reply:
I've removed the `<span class="inline-block">` as this was originally to improve the wrapping on smaller screens, however in retrospect it's not really an improvement 🙂 . 

We're now only wrapping `with` to make it lowercase.